code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -17,7 +17,7 @@
 	new /obj/item/radio/headset/heads/captain/alt(src)
 	new /obj/item/radio/headset/heads/captain(src)
 	new /obj/item/storage/belt/sabre(src)
-	new /obj/item/storage/box/gunset/pdh(src) // SKYRAT EDIT CHANGE - ORIGINAL: new /obj/item/gun/energy/e_gun(src)
+	new /obj/item/storage/box/gunset/pdh(src)//BUBBERSTATION EDIT TAKE 2
```

Comment:
I'm guessing you changed this in a previous PR & are only now updating the comment?

Reply:
Yeah take 2, they had PDH, then they didn't, then they did again.